### PR TITLE
Update Syscoin Masternode Install Script to 4.2

### PR DESCRIPTION
**Pull request by bigpoppa-sys**

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -166,7 +99,7 @@ install_sentinel(){
   else
     cd sentinel
     git fetch
-    git checkout master --quiet
+    git checkout dev-4.x --quiet
```
Comment (sidhujag) [inline script.sh:102]: should be master

File: script.sh [hunk 2]
```diff
@@ -454,11 +366,11 @@ create_and_configure_syscoin_user(){
   grep -q "alias syscoin-cli" ~/.bashrc || echo "alias syscoin-cli='syscli'" >> ~/.bashrc
   grep -q "sysd\(\)" ~/.bashrc || echo "sysd() { sudo su -c \"syscoind \$*\" syscoin; }" >> ~/.bashrc
   grep -q "alias syscoind" ~/.bashrc || echo "alias syscoind='sysd'" >> ~/.bashrc
-  grep -q "sysmasternode\(\)" ~/.bashrc || echo "sysmasternode() { bash <(curl -sL https://raw.githubusercontent.com/Syscoin/Masternode-install-script/master/script.sh) ; }" >> ~/.bashrc
+  grep -q "sysmasternode\(\)" ~/.bashrc || echo "sysmasternode() { bash <(curl -sL https://raw.githubusercontent.com/bigpoppa-sys/Masternode-Install-Script/master/script.sh) ; }" >> ~/.bashrc
```
Comment (sidhujag) [inline script.sh:369]: should point to Syscoin domain

File: script.sh [hunk 3]
```diff
@@ -9,18 +9,21 @@ fi
 HBAR="---------------------------------------------------------------------------------------"
 
 # import messages
-source <(curl -sL https://raw.githubusercontent.com/syscoin/Masternode-Install-Script/master/messages.sh) 
+source <(curl -sL https://raw.githubusercontent.com/bigpoppa-sys/Masternode-Install-Script/master/messages.sh)
```
Comment (sidhujag) [inline script.sh:12]: should be syscoin domain

File: script.sh [hunk 4]
```diff
@@ -209,7 +142,7 @@ configure_sentinel(){
 
   # setup cron for syscoin user
   sudo crontab -r -u syscoin
-  sudo crontab -l -u syscoin | grep sentinel-ping || echo "*/5 * * * * /usr/local/bin/sentinel-ping" | sudo crontab -u syscoin -
+  sudo crontab -l -u syscoin | grep sentinel-ping || echo "* * * * * /usr/local/bin/sentinel-ping" | sudo crontab -u syscoin -
```
Comment (sidhujag) [inline script.sh:145]: what is sentinel-ping?

File: script.sh (hunk 3 above)
Comment (bigpoppa-sys) [reply script.sh]: fixed

File: script.sh (hunk 4 above)
Comment (bigpoppa-sys) [reply script.sh:145]: left overs from 4.0 but isn't included - just file name

File: script.sh (hunk 2 above)
Comment (bigpoppa-sys) [reply script.sh]: fixed

File: script.sh (hunk 1 above)
Comment (bigpoppa-sys) [reply script.sh]: updated to master

**Pull request merged**